components/clp-mcp-server/clp_mcp_server/server/utils.py

```diff
@@ -83,8 +83,9 @@ def format_query_results(query_results: list[dict[str, Any]]) -> list[str]:
     kv-pairs:
     - "timestamp": An integer representing the epoch timestamp in milliseconds.
     - "message": A string representing the log message.
+    - "link": A link to open the log viewer displaying the message (optional).
```

Comment:
How about this to be consistent with other fields:
```suggestion
    - "link": A string representing the link to open the log viewer displaying the message (optional).
```